OBAKit/Models/unmanaged/OBAReferencesV2.h

```diff
@@ -17,6 +17,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void) addAgency:(OBAAgencyV2*)agency;
 - (OBAAgencyV2*) getAgencyForId:(NSString*)agencyId;
+- (NSMutableDictionary*) getAllAgencies;
```

Comment:
This class shouldn't publicly vend a mutable dictionary. I'd like to see the method signature changed to `NSDictionary`